src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -13,45 +13,45 @@
   public static final String AD_DRIVER_FORCED_STATUS_SUCCESS_SCRIPT =
       "top.window.adDriver2ForcedStatus['%s'.split(',')[0]]='success';";
   //SLOTS NAMES
-  public static final String BOTTOM_LB = "BOTTOM_LEADERBOARD";
+  public static final String BOTTOM_LB = "bottom_leaderboard";
   public static final String MOBILE_BOTTOM_LB = "bottom_leaderboard";
-  public static final String TOP_LB = "TOP_LEADERBOARD";
-  public static final String HIVI_TOP_LB = "top_leaderboard"; // revert to hivi_leaderboard after ae3 rollout to XX
-  public static final String TOP_BOXAD = "TOP_BOXAD";
-  public static final String FLOATING_MEDREC = "INCONTENT_BOXAD_1";
+  public static final String TOP_LB = "top_leaderboard";
+  public static final String HIVI_TOP_LB = "hivi_leaderboard";
+  public static final String TOP_BOXAD = "top_boxad";
+  public static final String FLOATING_MEDREC = "incontent_boxad_1";
   public static final String WIKIA_BAR = "WIKIA_BAR_BOXAD_1";
   public static final String MOBILE_TOP_LB = "top_leaderboard";
   public static final String MOBILE_AD_IN_CONTENT = "incontent_boxad_1";
   public static final String MOBILE_AD_IN_CONTENT_PLAYER = "incontent_player";
   public static final String MOBILE_PREFOOTER = "mobile_prefooter";
-  public static final String INCONTENT_PLAYER = "INCONTENT_PLAYER";
-  public static final String INVISIBLE_SKIN = "INVISIBLE_SKIN";
-  public static final String INVISIBLE_HIGH_IMPACT_2 = "INVISIBLE_HIGH_IMPACT_2";
+  public static final String INCONTENT_PLAYER = "incontent_player";
+  public static final String INVISIBLE_SKIN = "invisible_skin";
+  public static final String INVISIBLE_HIGH_IMPACT_2 = "invisible_high_impact_2";
   //HashMap with slots selector
   private static final Map<String, String> SLOTS_SELECTORS;
   private static final Map<String, String> SPECIAL_PAGE_SLOTS_SELECTORS;
   private static final Map<String, String> FILE_PAGE_SLOTS_SELECTORS;
   private static final Map<String, String> MOBILE_SLOTS_SELECTORS;
   private static final Map<String, String> SLOTS_TRIGGERS;
-
+  // Some changes are temporary due to ae2-ae3 switch
   static {
     SPECIAL_PAGE_SLOTS_SELECTORS = new HashMap<>();
-    SPECIAL_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
+    SPECIAL_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#top_leaderboard");
 
     FILE_PAGE_SLOTS_SELECTORS = new HashMap<>();
-    FILE_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
-    FILE_PAGE_SLOTS_SELECTORS.put(TOP_BOXAD, "#TOP_BOXAD");
-    FILE_PAGE_SLOTS_SELECTORS.put(BOTTOM_LB, "#BOTTOM_LEADERBOARD");
+    FILE_PAGE_SLOTS_SELECTORS.put(TOP_LB, "#top_leaderboard");
+    FILE_PAGE_SLOTS_SELECTORS.put(TOP_BOXAD, "#top_boxad");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#top_boxad" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)